internal/sequencers/avail/avail_listener.go

```diff
@@ -32,24 +31,29 @@ const (
 	DELAY                   = 500
 	ONE_SECOND_IN_MS        = 1000
 	FIVE_MINUTES            = 300
+	// FIVE_MINUTES = 30
 )
 
 type AvailListener struct {
-	FromBlock       uint64
+	AvailFromBlock  uint64
 	InputRepository *cRepos.InputRepository
 	InputterWorker  *inputter.InputterWorker
 	PaioDecoder     PaioDecoder
+	L1CurrentBlock  uint64
 }
 
 type PaioDecoder interface {
 	DecodePaioBatch(bytes string) (string, error)
 }
 
-func NewAvailListener(fromBlock uint64, repository *cRepos.InputRepository, w *inputter.InputterWorker) supervisor.Worker {
+func NewAvailListener(availFromBlock uint64, repository *cRepos.InputRepository, w *inputter.InputterWorker, fromBlock uint64) supervisor.Worker {
+	paioDecoder := ZzzHuiDecoder{}
```

Comment:
haha 😂

Reply:
@fabiooshiro He likes making tributes, like 'brunodo,' where 'bru' is part of my name.

Reply:
Love that one too. Brunodo, its perfect LOL